src/client/pages/SignInSuccess.tsx

```diff
@@ -1,233 +1,99 @@
-    overflow: hidden;
-  }
-
-  ${from.desktop} {
-    order: 2;
-  }
-`;
-
-const envelopeImage = css`
-  width: 220px;
-  margin-bottom: -26px;
-`;
-
-const button = css`
-  margin-top: ${space[4]}px;
-`;
-
 type SignInSuccessProps = {
   geolocation?: GeoLocation;
   consents: Consent[];
 };
 
-export const SignInSuccess = ({ consents }: SignInSuccessProps) => {
-  const { id, name, description, consented } = consents[0];
+export const SignInSuccess = ({
+  geolocation,
+  consents,
+}: SignInSuccessProps) => {
+  const autoRow = getAutoRow(1, gridItemColumnConsents);
   const clientState: ClientState = useContext(ClientStateContext);
   const { pageData = {}, queryParams } = clientState;
-  const { page = '', geolocation } = pageData;
-  const renderForm = () => (
-    <form
-      css={formStyles}
-      action={buildUrlWithQueryParams('/signin/success', {}, queryParams)}
-      method="post"
-      onSubmit={({ target: form }) => {
-        onboardingFormSubmitOphanTracking(
-          page,
-          pageData,
-          // have to explicitly type as HTMLFormElement as typescript can't infer type of the event.target
-          form as HTMLFormElement,
-        );
-      }}
-    >
-      <CsrfFormField />
-      <h2 css={consentTitle}>{name}</h2>
-      <p css={consentDescription}>{description}</p>
-      <Checkbox
-        name={id}
-        value={id}
-        label="Yes, sign me up"
-        defaultChecked={!!consented}
-      />
-      <Button type="submit" cssOverrides={button}>
-        Continue to The Guardian
-      </Button>
-    </form>
-  );
 
   return (
     <>
       <Header geolocation={geolocation} />
-      <main>
-        <div css={titleWrapper}>
-          <div css={[autoMargin, gridRow]}>
-            <h1 css={title}>Get the latest offers sent to your inbox</h1>
-          </div>
-        </div>
-        <div css={formBackground}>
-          <div css={[autoMargin, gridRowPadding]}>
-            <div css={formWrapper}>
-              <div css={[innerGridRow, formGridRow]}>
-                <div css={envelopeImageWrapper}>
-                  <EnvelopeImage invertColors cssOverrides={envelopeImage} />
-                </div>
-                {renderForm()}
-              </div>
+      <main css={mainStyles}>
+        <form
+          css={[autoRow()]}
+          action={buildUrlWithQueryParams('/signin/success', {}, queryParams)}
+          method="post"
+          onSubmit={({ target: form }) => {
+            onboardingFormSubmitOphanTracking(
+              'signin/success',
+              pageData,
+              // have to explicitly type as HTMLFormElement as typescript can't infer type of the event.target
+              form as HTMLFormElement,
+            );
+          }}
+        >
+          <CsrfFormField />
+          <ConsentsContent>
+            <h1 css={[heading, autoRow()]}>
+              Get the latest offers sent to your inbox
+            </h1>
+            <div css={[autoRow()]}>
```

Comment:
```suggestion
            <div css={autoRow()}>
```

Reply:
This is odd, this diff isn't showing as outdated, but this definitely isn't the most up to date code?

If you ever have need to check for usages along these lines, here's a regex `css=\{\[[a-zA-z]+?\]\}`. Seems there's only four times we do this elsewhere in the codebase.